### PR TITLE
FSA22V2-321: Fix style problems on home page

### PR DESCRIPTION
**Pull request by marissahuysentruyt**
### Description:

Some minor style fixes were implemented to continue to refine and match the homepage designs. This PR includes: 
- Refactoring the grid columns & rows to be a consistent size (using `fr` and `repeat`)
- Add an additional 2-column grid layout to accommodate more "medium" sized screens
- Use `:focus:not(:focus-visible)` on the component cards to prevent them from rotating when you click on them, but not when you tab to them
- Because the 1.4 value seemed too large, use `calc` to adjust the line-heights of all `h1` headings. 

### Spec:

See Story: [FSA22V2-321](https://sparkbox.atlassian.net/browse/FSA22V2-321)

### Validation:

- [x] This PR has code changes, and our linters still pass.
- [x] This PR affects production code, so it was browser tested (see below).

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down all related branches.
3. Run `npm install` to make sure you have all proper dependencies. (this project requires Node 16+)
4. Copy the `.env.example` file, making sure it's in the root of your project, and rename it `.env.local`. Search for "Accessible Components" in 1Password to find the API Key and Base ID. Add both of those secrets to your newly created `.env.local` file. That should establish your connection to our Airtable database.
5. Run `npm run lint` to confirm no errors.
6. Run `npm run test` to confirm all tests pass.
7. In terminal: `npm run dev`.
8. Check [localhost:3000](http://localhost:3000/) to see changes.
9. Run axe Devtools on affected pages to confirm no urgent errors.
10. On the homepage, resize the browser to see the grid change between a 1-, 2- and 3-column layout.
11. Tab through the homepage, until you get to your component of choice. Your tabbed component should be rotated slightly. Keeping that tabbed focus, with your cursor, click on a different component. When you click, you should not see that clicked component rotate. 

---

#### Browser Testing

In these browsers, behavior & design closely match original specifications. A user is able to access all content and functionality, including the usability of required assistive devices, such as keyboard and screenreader.

**macOS**

- [x] Safari (last 2 major versions)
- [x] Chrome (last 6 months)
- [x] Firefox (last 6 months)

**Windows**

- [x] Chrome (last 6 months)
- [x] Firefox (last 6 months)
- [x] Edge (last 6 months)

**Mobile**

- [x] Safari (last 2 major versions)
- [x] Chrome on Android (last 6 months)
- [x] Firefox on Android (last 6 months)

### CODE REVIEW

File: src/styles/components/_component-card.scss [hunk 1]
```diff
@@ -86,11 +86,18 @@
...
+  &:focus-visible .component-card {
     position: relative;
     z-index: 1;
     outline: 2px solid var(--accent-color);
     outline-offset: 2px;
     transform: rotate(2deg);
   }
+
+  // click focus has no rotation
+  &:focus:not(:focus-visible) .component-card {
```
Comment (rkelbel) [inline src/styles/components/_component-card.scss:99]: Great use of comments for documenting what these are doing. 🙂

File: src/styles/components/_component-card.scss [hunk 2]
```diff
@@ -86,11 +86,18 @@
...
+    outline: 2px solid var(--accent-color);
+    outline-offset: 2px;
+  }
 }
```
Comment (rkelbel) [inline src/styles/components/_component-card.scss:103]: > Keeping that tabbed focus, with your cursor, click on a different component. When you click, you should not see the component rotate.

Based on the validation instructions I did as instructed, when I focus with the keyboard on a component but then click on another component with the cursor it does rotate back.  Did you mean the original focused component shouldn't rotate back or the component that is clicked on should not rotate?

Is this the correct behavior you needed us to test?
component focus click

**Review by rkelbel — COMMENTED** [review]
Mobile and Tablet views look awesome!

File: src/styles/components/_component-card.scss (hunk 2 above)
Comment (marissahuysentruyt) [reply src/styles/components/_component-card.scss:103]: Yes! Before the mouseclick was _also_ rotating the cards. We wanted to get rid of that. I think it's ok that the tabbed focus un-rotates.

Comment (rkelbel) [reply src/styles/components/_component-card.scss:103]: I think it is too, I just wanted to verify I understood which one should be moving or not moving on click. lol

File: src/styles/layout/_component-grid.scss [hunk 3]
```diff
@@ -56,7 +56,19 @@ $breakpoint-accordion-medium-550: 34rem;
 
     @media (min-width: $breakpoint-scale-up) {
       display: grid;
-      list-style: none;
+      grid-template-rows: repeat(4, 1fr);
```
Comment (jkohrt7) [inline src/styles/layout/_component-grid.scss]: We should reduce the height a little here with `grid-template-rows: repeat(3, minmax(0, 0.8fr));`.

**Review by jkohrt7 — CHANGES_REQUESTED** [review]
The fr units were making grid elements rather tall. Besides that, everything works smoothly and passes the checks!

File: src/styles/layout/_component-grid.scss (hunk 3 above)
Comment (marissahuysentruyt) [reply src/styles/layout/_component-grid.scss]: Consider it done! https://github.com/sparkbox/accessible-components/pull/137/commits/4297d2dfc667f760eaa5d595a8df506453f3945c

File: src/styles/layout/_component-grid.scss [hunk 4]
```diff
@@ -56,7 +56,19 @@ $breakpoint-accordion-medium-550: 34rem;
...
+        "accordion disclosure"
+        " tabs tabs"
+        "max-scott max-scott";
+    }
+
+    @media (min-width: $breakpoint-grid-large-screen-1115) {
+      grid-template-columns: unset;
+      grid-template-columns: 1fr 1fr 1fr;
+      grid-template-rows: unset;
+      grid-template-rows: 1fr 1fr 1fr;
```
[the remark this answers is not in the document]
Comment (marissahuysentruyt) [reply src/styles/layout/_component-grid.scss]: Done! Thanks for walking me through how this was supposed to work! 

https://github.com/sparkbox/accessible-components/pull/137/commits/4297d2dfc667f760eaa5d595a8df506453f3945c

**Review by jkohrt7 — APPROVED** [review]
Thanks for making those changes!

**Review by rkelbel — APPROVED** [review]
Looks great! Nice work Marissa!

thumbs up

**Pull request merged**